sdk/personalizer/ai-personalizer-rest/src/models.ts

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why is there no `generated` folder?

Reply:
Since the generated code is what is public and there is no other layer that hides generated code, it made sense to have this not be in a generated folder and just directly in src.